examples/basics/custom-type/custom-type.go

```diff
@@ -25,17 +29,24 @@ type (
 )
 
 func main() {
-	// Connect to a local Qlik Associative Engine
+	// Fetch the QCS_HOST and QCS_API_KEY from the environment variables
+	QCS_HOST := os.Getenv("QCS_HOST")
+	QCS_API_KEY := os.Getenv("QCS_API_KEY")
+
+	// Connect to Qlik Cloud tenant
 	ctx := context.Background()
-	global, err := enigma.Dialer{}.Dial(ctx, "ws://localhost:9076", nil)
+	rand.Seed(time.Now().UnixNano())
```

Comment:
Why seed? Is there a collision problem for the session apps?

Reply:
well, yes exactly - still not sure why this happens - the tests are running in different parallel workflows `rand.Int()` should have been different for all cases.

Reply:
You could use the `time.Now().UnixNano()` instead of `rand.Int()` though. If the timestamp is the same the number generated should be the same anyway right?

Reply:
Actually, it says on the documentation of [rand.Seed](https://pkg.go.dev/math/rand#Seed) that
> ...if not called it will be as calling rand.Seed(1)...

Which in turn means the first number will always be the same, news to me!
Example: https://go.dev/play/p/xthPfRzcbD_m